src/test/java/com/example/javaitmo2/web/CarControllerTests.java

```diff
@@ -81,7 +84,9 @@ public void shouldReturnedSuccessGetList() throws Exception {
 
     @Test
     public void shouldReturnedSuccessGetOne() throws Exception {
-        CarRequest request = new CarRequest("RRR12TTFGHJS89", "toyota", 5, null);
```

Comment:
extract as local variable, expectedVin for example. And by the way in asserEquals (expected, testResult), so flip it

Reply:
+